### PR TITLE
Cache GUI localization and UI changes

### PR DESCRIPTION
**Pull request by andreeis**
Make sure all the strings in the cache GUI are localized.
Change the UI appearance so that the web view looks more similar with CppTools.
I was not able so far to change the following:
- the active border (the border when any UI element is in focus). For CppTools is blue, for CMake Tools is a yellow/orange but I tried to change it in various ways without success
- the checkboxes have a blue background when set and remain in sync with the current theme when not set. CppTools checkboxes are in sync with the current theme also when set.

### CODE REVIEW

File: src/cache-view.ts [hunk 1]
```diff
@@ -161,46 +243,83 @@ export class ConfigurationWebview {
...
+              contrastBorder: var(--vscode-focusBorder);
+              contrastActiveBorder: var(--vscode-focusBorder);
           }
         </style>
         <script>
           const vscode = acquireVsCodeApi();
           function toggleKey(id) {
             const label = document.getElementById('LABEL_' + id);
-            label.textContent = label.textContent == 'ON' ? 'OFF' : 'ON';
+            label.textContent = label.textContent == '${onButtonText}' ? '${offButtonText}' : '${onButtonText}';
```
Comment (bobbrow) [inline src/cache-view.ts:322]: I think these are CMake enum values.  I don't believe they should be localized.

File: src/cache-view.ts [hunk 2]
```diff
@@ -40,6 +41,16 @@ export class ConfigurationWebview {
     );
   }
 
+  async persistCacheEntries() {
```
Comment (bobbrow) [inline src/cache-view.ts:44]: This seems to save the temporary changes directly into the cache. I thought that you were going to persist them in memory and then reload the changes the next time the page renders. I don't think we should be saving stuff without the user pushing the "save" button.

Comment (andreeis) [reply src/cache-view.ts:44]: I agree. I noticed that the settings get persisted in settings.json when changing focus or closing the tab so I thought to be matching that but then the SAVE button has no reason to exist.

File: src/cache-view.ts (hunk 1 above)
Comment (andreeis) [reply src/cache-view.ts:310]: Ups, yes

**Review by bobbrow — APPROVED** [review]

**Pull request merged**